codespell_lib/data/dictionary.txt

```diff
@@ -9544,6 +9592,7 @@ meterologist->meteorologist
 meterology->meteorology
 methaphor->metaphor
 methaphors->metaphors
+methds->methods
```

Comment:
please also add `methd->method` above this, thx

Reply:
Done